conf/nginx.conf

```diff
@@ -20,6 +20,8 @@ location ^~ #LOCATION# {
   add_header X-Robots-Tag none;
   add_header X-Download-Options noopen;
   add_header X-Permitted-Cross-Domain-Policies none;
+  # Add data: to allow /ynhpanel.css to be load due to image on data:base64
+  more_set_headers Content-Security-Policy "default-src data:;";
```

Comment:
here wouldn't it be safer by adding `'self'` directive: `default-src 'self' data:;`?